test/recompilation-tests/with-active-venv/check.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
No need to test in separate environments (like active venv, default etc.), default is enough. The content of check.sh should be split into multiple test cases.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/106/commits/eb50f395bf7853df489c6e38fde504c92cb3bc78